src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -3203,4 +3203,16 @@ Zero-width positive lookbehind assertions are typically used at the beginning of
   <data name="You_must_rename_an_identifier" xml:space="preserve">
     <value>You must rename an identifier.</value>
   </data>
+  <data name="This_0_has_1_references" xml:space="preserve">
+    <value>This {0} has {1} reference(s).</value>
+  </data>
+  <data name="_0_reference_unquoted" xml:space="preserve">
+    <value>{0} reference</value>
```

Comment:
why the `"unquoted"`?

Reply:
To distinguish it from the string also named "_0_reference" but has the reference number in quotes (e.g. `'5' references` instead of what we want for codelens, which is `5 references`